types/node-jose/index.d.ts

```diff
@@ -97,6 +97,7 @@ export namespace JWE {
         fields?: object;
         contentAlg?: string;
         protect?: string | string[];
+        iv?: Buffer;
```

Comment:
Can this be a `string`? As per source you've linked:
https://github.com/cisco/node-jose/blob/1f86e23fc6/lib/jwe/encrypt.js#L391-L395

Reply:
Thanks for the review. With https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53788/commits/d2159fb5453a5c1d02f836f14cfc900985cad77f, I've made the following changes:

* added `string` and a test case for it
* clarified the use case for [`Buffer`](https://nodejs.org/api/crypto.html#crypto_crypto_randombytes_size_callback) in the test case